### ISSUE TITLE
New sidebar design does not make it clear when you don't have permission to invite

### ISSUE THREAD

**Issue by gabrc52** [vector-im/element-web#26586]
### Steps to reproduce

1. Go to a room where you don't have permission to invite
2. On the right sidebar, click on "People"

#### What did you expect?

The button is clearly disabled (greyed out, faded, etc), or at the very I get an error message saying that I don't have permission to invite when I click on it.  

The button should look different from the way it looks like when you do have permission, like before the redesign (not saying it should look like it looked before the redesign, but there should be a clear visual distinction that you cannot invite).

#### What happened instead?

Pressing the button does nothing, leaving me with the impression that Element is broken rather than a permissions issue.

[Screencast from 2023-11-15 02-12-17.webm](https://github.com/vector-im/element-web/assets/24363938/5fb056f8-4119-4d89-b637-2d8db6e59ea0)

Only after looking more carefully, I realize that the tooltip says I have no permission to invite.

![image](https://github.com/vector-im/element-web/assets/24363938/ff7a45fb-6269-45ee-9de8-ec885cab57d6)

### Application version

Element version: 1.11.48

### Will you send logs?

No
